### ISSUE TITLE
Why was this abandoned?

### ISSUE THREAD

**Issue by joaomilho** [rawrmaan/restyped#14]

**Comment by bobmoff** [comment]
Yes, I wonder too. I am looking for something exactly like this. I will try using it but it feels a little worrying that the last commit was so long ago.

**Comment by joaomilho** [comment]
I didn't really work for me without minor adjustments. I'm trying to find the closest thing to https://www.servant.dev/ in TS.

**Comment by rawrmaan** [comment]
Hey guys, thanks for the concerns. @joaomilho wondering what types of changes you had to make? Thanks!
